### ISSUE TITLE
[MDEP-388] NPE in analyze-report

### ISSUE THREAD

**Issue by jira-importer** [apache/maven-dependency-plugin#827]
**[Benson Margulies](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=bmargulies)** opened **[MDEP-388](https://issues.apache.org/jira/browse/MDEP-388?redirect=false)** and commented

Using Maven 3.0.4, and site plugin 3.2, and an execution of the analyze-report goal, I get the following NPE. I'm going to go try to make some sense and add notes here.

```
Caused by: java.lang.NullPointerException
	at org.apache.maven.plugin.dependency.AnalyzeReportMojo.executeReport(AnalyzeReportMojo.java:100)
	at org.apache.maven.reporting.AbstractMavenReport.generate(AbstractMavenReport.java:135)
	at org.apache.maven.plugins.site.ReportDocumentRenderer.renderDocument(ReportDocumentRenderer.java:228)
	at org.apache.maven.doxia.siterenderer.DefaultSiteRenderer.renderModule(DefaultSiteRenderer.java:319)
	at org.apache.maven.doxia.siterenderer.DefaultSiteRenderer.render(DefaultSiteRenderer.java:135)
	at org.apache.maven.plugins.site.SiteMojo.renderLocale(SiteMojo.java:175)
	at org.apache.maven.plugins.site.SiteMojo.execute(SiteMojo.java:138)
	at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:101)
	... 20 more
```

---

**Affects:** 2.5.1

**Issue Links:**
- [MDEP-366](https://issues.apache.org/jira/browse/MDEP-366) NPE when running site
 (_**"duplicates"**_)

**Comment by jira-importer** [comment]
**[Benson Margulies](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=bmargulies)** commented

the supposedly-injected 'project' field is null. Just in case this was pirate error:

```
<plugin>
              <artifactId>maven-dependency-plugin</artifactId>
              <version>2.5.1</version>
               <reportSets>
                 <reportSet>
                   <reports>
                     <report>analyze-report</report>
                   </reports>
                 </reportSet>
               </reportSets>
          </plugin>
```

**Comment by jira-importer** [comment]
**[Rene Reitmann](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=rreitmann)** commented

I am running into the same NPE. I am using a multi-module project with Maven 3.0.4 and maven-site-plugin 3.2. Is there any workaround?

**Comment by jira-importer** [comment]
**[Arnaud Heritier](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=aheritier)** commented

This is fixed in the incoming 2.6 release
